### ISSUE TITLE
FO4Edit: EXPL with projectiles

### ISSUE THREAD

**Issue by smangham** [TES5Edit/TES5Edit#328]
Currently, an unmodified copy-as-override of the MIRV mininuke explosion into a file results in the split failing, even if it is the only change in the only mod active.

I've attempted to make other explosions with placed projectiles and none have worked; presumably they're all missing something.

**Comment by fireundubh** [comment]
What's the Form ID of the affected record? Which version of FO4Edit are you using?

In the latest build, all EXPL records copy as overrides without errors.

**Comment by smangham** [comment]
FO4Edit 3.1.3 - 1b8db5e, 7/1/16 version from the Nexus.
The formID is 01C6BFF. The file in question is here: https://www.mediafire.com/?46jcwce4e2jszem

**Comment by fireundubh** [comment]
That record copies as an override without errors in that version of FO4Edit.

What are you trying to do exactly?

**Comment by smangham** [comment]
Weird. Are you on the beta patch? Maybe it's not a FO4Edit problem and the patch has fixed it. 
Could you try the .esp I uploaded to see if it works on your machine?

I was trying to make more modifications that added cluster bombs by copying the MIRV explosion form (and OMOD) and changing the projectiles. That didn't work, so I stepped back through it until I found that even a perfect copy of the MIRV explosion as a new form didn't work, then I tested just copy-as-overriding it.

**Comment by fireundubh** [comment]
> That didn't work, so I stepped back through it until I found that even a perfect copy of the MIRV explosion as a new form didn't work

You're still not being very clear. If you are reporting an issue with xEdit:

* What didn't work?
* How did what not work?
* What were the steps you used to do whatever you were doing?
* What was the result?
* What was the result you expected?

I've otherwise found no errors with regard to the copying of the record.

> I was trying to make more modifications that added cluster bombs by copying the MIRV explosion form (and OMOD) and changing the projectiles. 

We can't really help you with the creation of mods here. I don't know how familiar you are with Bethesda's systems, but in this case, you should understand the relationships between the WEAP, OMOD, PROJ, and EXPL records, and what these records govern. I'd recommend building references in xEdit, using the Referenced By tab to follow the relationships, and Ctrl+Click Form IDs to learn what each record does.

**Comment by smangham** [comment]
I am familiar with how the relationships are set up. I described the mod I was trying to make as I thought you wanted background detail.

*What didn't work:*  Any EXPL record with a 'spawn projectile' entry edited by FO4Edit.
*How did it not work:* On explosion, they did not spawn projectiles.
*What were the steps/result/expected result:* 
*Steps 1:* I copied the MIRV explosion, changed only the spawned projectile, then copied the MIRV cluster PROJ and changed the explosion to the altered MIRV explosion. I then copied the MIRV OMOD and altered it to use the new MIRV-exploding PROJ. 
*Result 1:* When applying this OMOD to a weapon, the explosion did not generate projectiles. 
*Expected 1:* The explosion would generate projectiles.

*Steps 2:* I then altered back the projectiles in my MIRV explosion to the normal MIRV child projectiles. My MIRV explosion now perfectly matched the unmodified MIRV.
*Result 2:* It still didn't generate projectiles. Even if it was the only mod active (and the only OMOD on the Fat Man).

*Steps 3:* As the setup for this OMOD was now identical to the normal MIRV, I assumed it must be an issue with missing info on the EXPL record. I made a new file and copied-as-override the MIRV explosion into a new file with no other modifications.
*Result 3:* Activating **only** that file resulted in the explosion not spawning projectiles.
